### PR TITLE
William/fix send logs

### PR DESCRIPTION
**Pull request by swansontec**
### CHANGELOG

 none

### Dependencies

 none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [ ] Tested on iOS device
- [ ] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204108239274817

### CODE REVIEW

File: src/components/modals/WalletListMenuModal.tsx [hunk 1]
```diff
@@ -210,24 +203,19 @@ export function WalletListMenuModal(props: Props) {
...
-        </ScrollView>
-      </View>
-      <LinearGradient style={styles.modalCloseButton} colors={[xButtonTopColor, xButtonBottomColor]} start={xButtonGradientStart} end={xButtonGradientEnd}>
-        <ModalCloseArrow onPress={handleCancel} />
-      </LinearGradient>
+      <ModalScrollArea onCancel={handleCancel}>
+        {options.map((option: Option) => (
+          <TouchableOpacity key={option.value} onPress={() => optionAction(option.value)} style={styles.row}>
+            <AntDesignIcon
+              // @ts-expect-error
```
Comment (Jon-edge) [inline src/components/modals/WalletListMenuModal.tsx:210]: Can this be fixed?

File: src/components/modals/WalletListMenuModal.tsx [hunk 2]
```diff
@@ -210,24 +203,19 @@ export function WalletListMenuModal(props: Props) {
...
+              name={icons[option.value] ?? 'arrowsalt'} // for split keys like splitBCH, splitETH, etc.
+              size={theme.rem(1)}
+              style={option.value === 'delete' ? [styles.optionIcon, styles.warningColor] : styles.optionIcon}
+            />
+            <Text style={option.value === 'delete' ? [styles.optionText, styles.warningColor] : styles.optionText}>{option.label}</Text>
```
Comment (Jon-edge) [inline src/components/modals/WalletListMenuModal.tsx:215]: EdgeText?

**Review by Jon-edge — CHANGES_REQUESTED** [review]

File: src/components/modals/WalletListMenuModal.tsx (hunk 1 above)
Comment (swansontec) [reply src/components/modals/WalletListMenuModal.tsx:210]: Look at the PR with whitespace hidden. It's been this way for years, this PR is not about fixing it.

File: src/components/modals/WalletListMenuModal.tsx (hunk 2 above)
Comment (swansontec) [reply src/components/modals/WalletListMenuModal.tsx:215]: Look at the PR with whitespace hidden. It's been this way for years, this PR is not about fixing it.

**Review by Jon-edge — APPROVED** [review]

**Pull request merged**